src/app/pages/questions/services/audio-record.service.ts

```diff
@@ -19,16 +21,31 @@ export class AudioRecordService {
   isRecording: boolean
   encoding = 'base64'
   data: any
+  private audioSamplingRate: number = DefaultAudioRecordOptions.SampleRate
 
   constructor(
-    private logger: LogService
-  ) {}
+    private logger: LogService,
+    private remoteConfig: RemoteConfigService
+  ) {
+    this.remoteConfig
+      .subject()
+      .subscribe(config =>
+        config
+          .getOrDefault(
+            ConfigKeys.AUDIO_SAMPLING_RATE,
+            String(this.audioSamplingRate)
+          )
+          .then(rate => (this.audioSamplingRate = Number(rate)))
+      )
+  }
 
   startAudioRecording(): Promise<any> {
     return new Promise((resolve, reject) => {
       return VoiceRecorder.requestAudioRecordingPermission().then(
         (result: GenericResponse) => {
-          return VoiceRecorder.startRecording()
+          return VoiceRecorder.startRecordingWithCompression({
+            sampleRate: this.audioSamplingRate
```

Comment:
What about other parameters? I know they are the same in both versions, but they might change in the future, so it would be good if the app specified those too.

Reply:
Yup made the changes. Plugin updates to allow configuration is here as well: https://github.com/mpgxvii/capacitor-voice-recorder/pull/2